.github/workflows/verify_newproject_script.yml

```diff
@@ -30,8 +30,11 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
-      - name: Verify generating new project from RxTemplate
-        run: ./newproject.sh -t rx -p co.myproject.example -n "MyProjectExampleRx"
-
       - name: Verify generating new project from CoroutineTemplate
-        run: ./newproject.sh -t crt -p co.myproject.example -n "MyProjectExampleCoroutine"
+        working-directory: scripts
+        run: |
+          curl -s "https://get.sdkman.io" | bash
+          source "/home/runner/.sdkman/bin/sdkman-init.sh"
+          sdk install kotlin
+          sdk install kscript
```

Comment:
I think we should separate this step into a new preparation step 🤔  It's also because we don't need to do these steps in `working-directory: scripts`

Reply:
Yeah, I tried something like this before. 

```
     - name: Install sdkman
        run: |
            curl -s "https://get.sdkman.io" | bash
            source "/home/runner/.sdkman/bin/sdkman-init.sh"

      - name: Install kotlin
        run: sdk install kotlin // At this step it would give an error, saying: "sdk command not found"

      - name: Install kscript
        run: sdk install kscript

      - name: Run script
        working-directory: scripts
        run: kscript new_project.kts package-name=co.myproject.example app-name="My Project"
```

Any idea what I was missing here? 🙇

Reply:
@Tuubz I think we can group all 4 preparation steps into 1

```
          curl -s "https://get.sdkman.io" | bash
          source "/home/runner/.sdkman/bin/sdkman-init.sh"
          sdk install kotlin
          sdk install kscript
```

Reply:
@luongvo I tried this before, but then it wasn't able to find the kscript command

So that's why I decided to group them together, any idea how to fix this? 🙇 

<img width="646" alt="Screen Shot 2565-06-07 at 10 08 14" src="https://user-images.githubusercontent.com/18277915/172287378-0e9dbefa-6817-47fa-89c5-092b0b5568e8.png">